packages/icons/scripts/generate-index.ts

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
+import type { MultiSourceIconSource } from "./fetch-svgs.ts";
+
+const GENERATED_HEADER = `/*
+* THIS FILE IS GENERATED. DO NOT EDIT IT.
+*/\n
+/* eslint-disable */`;
+
+export const generateIndex = (componentDirectory: string, iconsByNames: MultiSourceIconSource[]) => {
+    const iconList = iconsByNames.map(icon => icon.name + "Icon");
+    const indexFile = `${componentDirectory}/index.ts`;
+    const indexContent = `${GENERATED_HEADER}\n
+${Object.values(iconsByNames).map(icon => `export * from "./${icon.name}Icon.tsx";`).join("\n")}
+\nexport const iconNames = ${JSON.stringify(iconList)}`;
```

Comment:
i suggest adding "as const" at the end of the iconNames array. 
```
\nexport const iconNames = ${JSON.stringify(iconList)} as const;`;
```
This will allow better usage of this array in the tests, removing the need for `as unknown as ...`

Reply:
you can check what the difference between the 2 is in this typescript playground: https://www.typescriptlang.org/play?#code/PTAEEsGMHsDsDkCGBbApgZwp9AXATuLAOYDaAugFCoAeADtHjqDLLhC0mpgLygkBEAQQAmwgJIt+AGiGiAwogA2qWMMR4JcabOEBVdKg2SZg4soAi0AO6xNsbaaLKAMqgBmOOw7OoASuCIAC09jIR9dWi8TPEhA8AA3VCihPDxrSxtkwVTrVw8snKt-IJCtaLSrCIKK9AiM21CAIXV0ZOb0AGtUHAAjJUU2lRxoNuhFcSboaA7R6eR1GaaAV0VlUvsZRpXFAwBPJNCFZVV1ZIU0PEQz08P1bvrrvHvrWABlMfAJsv4FJ5wq25-CLvRSfa7rbRyQLqHDNIzfKEw5yEA4I6GMAAK4FRGx+gVQkA68zwizRhgIGDO4BiykcLncEJkcmpkFpPmKwTOimghLOY0UiFoBjyjJ+-MFBg5ork0GQaFg0tl8pwUIJpNxMtouz5SwVZ3Q8WS5lQaxx2nM4HQyEtrVCllWN2+9W5iC+uIAosJwKL3dRvcl3fsABKfYQqAP7ABqlvAPWUyQAYuBlKreaEk4ocIZEwkzTIEwKiImpr1+sWlvDcQBxRBLIh5-hV8D5UJVy60QLJNtg1sESKhIOyhtiWBuEahMQ4JRQZIAaVQfR6yWRJR6KyXoWRsHV2mcPJ3Mj3hNQwng0CzyQAsohk1eoGkO3AG5f79BH7BUBb0Ig4ye77Alltb5L2gRJkngVArBlOUhnAyCMyzSttAgqwmxbb4ULPC9Qkw89mygRAcHAOA4KsV4iMJbUcMggAVH9SJo1BkFoAVsIw2jwH7DDzwbLCFymA9+Cw-DIEI4iGm+AB5D9kkk2gVFea141CSS8DDJ5hGRXBZLwIhEFgcAAC8xJI0IMVrAxkgxYQ3Cs1BEEgBsMUMRy9TMwgrMIEEe2+DEBSo3z-O8t1tD8wCrLSIgnnQIDcQARSWDAiNM75fHsyBkok3E0sgCsCGIZI0rcaLO1CNLrVUbMyqY0CG3K2qjhUNQkJkerEn0KrUtQFiAuy7qGFFNKrEIELWv3bpkleeyYlK74poy8TJqapacCI4hYu0V50QbLa7lG-gtuxcZJoGybaAWZQNpkV5aDSJZRXIx1cUevBgsmqdGEkxIWoOiiOl6zb8HALocECO6gkmpYelwS4MshyBHJi5I6KLUJGJQZGmJYwiGxosHUFxwIlmQaGHjRomSdqLjcRozjkascdvl0WB1FiXNkmZ9TQRk0JmYYdSTy00UIgMVYOYMH7ozDRncSl1BoHOQwrlCOWFZQJWvx-ZR9oAdXUAyCskSgQHYOBOAwQR0BlVgmEtUBEFAUE2GgNx7dWUBQdQe34hvAVf1AGHCCICBYHt0P1EuXYqDoAbmDgNgoDN9X0Et622F4AQRH2rPGpOH6s46-OfDJ746VQEUsnZAJOVCMuAVLmb2drwoS9xbIKgr5uKilaprHrtvCip1vtDhK7+HaEG+jFpohhlkexn2xoBNmIkFjabYJuWVY9gbXPmuuC5lYRJ7ITuHBh6ZM-6jewFun70+gVoG-j+ldFYRPpk363Xe36xH+1WJIJVUqRsRj2ZDSVAZdO4IhZGyJwfhq7Sm5GmBE4ohTlwZHyB06Ce6HCVLBPBMEFSpiAW+AGTJoC6mlAaI0JpugNi-NaJGdpxQ-WdNAV0AYvQ+j9D6YMoZwyhEDKgaM6BYzKW+BmVAJCcyZk6riJMYF0yFmLOeKeAx0yUJ+jWOsDY0KijbIKWa1YCD7W7NTbQg40DJBHGOGxU5QSQDnAueim5EFrkUBub4W5BJHl8eNU8PEry+zvJAB8gQnyhPCU+TWv59ovgAmPECSj2JQXwW5VJCF5HIUgvo0ifF8l4TcARTKpFyJQH+gxVxqTGLMVYrxDiFiZAFJwjxHoy9WlEWKaJUpKkeZSXkm8JSDZVIC00paUUqk9IGWMr03yFknI2Tsg5JyLkCG+Q8u5N4Hx9p+UQOQ-gezdjP1xGFMeGJIrRTHglJKi1qoOTmX1XKIDDZdWKhgYx2hyojWya1GqKS+rIAakoJqH9+BtVQIXQq3VFAHLSvQRg0LhqqEKuNB601YhLQWilZ6y1QhTVWkHMeu0niTW2vtQ6JoKWnXxedEkl0zp3Qeh9d66gTmAxhF9X5v0KkHPIgQEGYNKEQ3xVDGGDz4aIzHijTGGM0ZY3qcjfGhNiakxeEq1VVNkZ0zRgzDmLNG4Au0JzQw3MGx8zUoYQWEyOboOnkzCWyRVZOs+PLRWlwXXS3dYgWJ2tkh6zwAbVGWgyD20wCwXAABuCgQA